modules/end-user-guide/nav.adoc

```diff
@@ -1,10 +1,10 @@
-.End-user guide
+.User guide
```

Comment:
Ideally, we should also rename the module and add redirections. We can do it in a subsequent pull request.

Reply:
hmm. I can do this now, may as well.

Reply:
done